akka-http-core/src/main/java/akka/http/javadsl/model/headers/HttpOriginRange.java

```diff
@@ -10,10 +10,10 @@
 /**
  * @see HttpOriginRanges for convenience access to often used values.
  */
-public abstract class HttpOriginRange {
-    public abstract boolean matches(HttpOrigin origin);
+public interface HttpOriginRange {
```

Comment:
Yes sadly this is breaking for all users of the method `matches`...
Since the invoke bytecode changes form invokevirtual to invokeinterface, so you get

```
Exception in thread "main" java.lang.IncompatibleClassChangeError: Found interface Example, but class was expected
```